mirrord-cli/src/config.rs

```diff
@@ -23,42 +23,19 @@ pub(super) enum Commands {
 #[clap(group(
     ArgGroup::new("exec")
         .required(true)
-        .args(&["target", "pod-name", "config-file"]),
+        .args(&["target", "config-file"]),
 ))]
 pub(super) struct ExecArgs {
     /// Target name to mirror.    
     /// Target can either be a deployment or a pod.
     /// Valid formats: deployment/name, pod/name, pod/name/container/name
-    #[clap(short, long, value_parser)]
+    #[clap(short = 't', long, value_parser)]
     pub target: Option<String>,
 
     /// Namespace of the pod to mirror. Defaults to "default".
     #[clap(long, value_parser)]
```

Comment:
```suggestion
    #[clap(short = 'n', long, value_parser)]
```

Reply:
why "n"? --target, or -t would make sense, wouldn't it?

Reply:
+1